core/unit_test/TestViewAPI.hpp

```diff
@@ -950,9 +950,9 @@ class TestViewAPI {
 
   static void run_test_mirror() {
     using view_type   = Kokkos::View<int, host>;
-    using mirror_type = typename view_type::HostMirror;
+    using mirror_type = typename view_type::host_mirror_type;
 
-    static_assert(std::is_same_v<typename view_type::HostMirror,
+    static_assert(std::is_same_v<typename view_type::host_mirror_type,
                                  typename view_type::host_mirror_type>);
```

Comment:
Ok, this answers my question ... you did a global search and replace.
Please revert and guard it by deprecated code 4 or remove

Reply:
AH good catch

Reply:
Thanks @JBludau, updated.